packages/components/src/handler.ts

```diff
@@ -487,7 +565,7 @@ export class AnalyticHandler {
                 await monitor.trackEvent('chain', 'start', {
                     runId,
                     name,
-                    userId: this.options.chatId,
+                    // userId: this.options.chatId,
```

Comment:
is it not possible to pass in `chatId` here for `trackEvent`? this `AnalyticHandler` is used when the module is not from langchain such as `OpenAIAssistant`

Reply:
Thing is chatId is different than the userId, so this will have unintended consequences in Lunary (showing 1 new user for each conversation). 

Ideally we'd instead pull the current lead and pass the lead ID here.

Reply:
got it! in that case, I suppose we remove the commented line?

Reply:
Done!